containers/build-or-download-cvmfs-debs.sh

```diff
@@ -5,7 +5,10 @@ apt-get update
 apt-get install -y wget
 if [ "$arch" = "ppc64el" ] || [ "$arch" = "arm64" ]
```

Comment:
There's a typo in `ppc64le` above.

Reply:
That's actually a Debian thing, really annoying:
https://wiki.debian.org/ppc64el

I was using $(uname -m) everywhere, which returns `aarch64`, `ppc64le`, and `x86_64`. But for Debian packages they, apparently, use `amd64`, `arm64`, and `ppc64el`. :-(

Reply:
```
Singularity> dpkg --print-architecture
ppc64el
```